app/views/events/show.html.erb

```diff
@@ -36,5 +36,6 @@
 </p>
 
 <%= link_to 'RSVP for this event', new_event_guest_path(@event) %>
+<%= link_to 'Volunteers', event_assignments_path(@event) %>
```

Comment:
this should only appear when you're logged in

Reply:
I'm not sure it makes sense to only have that available when logged in. If someone is not logged in and they click it, it takes them to the sign in page. This seems like desirable behavior to me, especially since there is currently not a sign in link otherwise.

Reply:
We don't want guests to see the ability to volunteer. once volunteers log in, they can see the link to volunteer.

Reply:
in general, there should be no way to get to a "log in" page unless you know the url to sign in.